packages/workspace/src/utilities/run-webpack.ts

```diff
@@ -26,7 +26,10 @@ export function runWebpack(config: any, webpack: any): Observable<any> {
     } else {
       webpackCompiler.run((err, stats) => {
         callback(err, stats);
-        subscriber.complete();
+
+        webpackCompiler.close(() => {
```

Comment:
Instead of monkey patching webpack above, perhaps it is easier to check for the existence of a `close` method?

```ts
if (webpackCompiler.close) {
  webpackCompiler.close(() => {
    subscriber.complete();
  });
} else {
  subscriber.complete();
}
```

Reply:
@FrozenPandaz, but if you check the close method in that case you will have a garbage after release of the Nx 13. If I do not mistake you will remove files with monkey patched code anyway. Thus it looks better in my opinion. Anyway it is up to you

Reply:
Thank you for thinking ahead!

That is okay because it is only 1 place. Please add a `TODO` comment saying to clean it up the conditional after Nx 13 :smile:.

Reply:
@FrozenPandaz, done. I removed the fallback for webpack 4 and added `if` statement with `TODO`.